### DISCUSSION TITLE
LBM appears to trigger Fortnite Bug [buttons get triggered Infinitly]

### DISCUSSION THREAD

**Discussion by Momo1352** [mgth/LittleBigMouse#266] [Q&A]
Yo, I know its a weird bug/request, but I noticed that whenever I play while running LBM some keys (like LEFTMB, RMB, interaction key) get triggered infinitly. So if I press LMB it sometimes continues to "press" the button until I press the same button again.
Also i remember i had the same kinda issue in Valorant, but not sure if this was bc of LMB.

Anyways thanks for the App, love it and use it everyday.

**Comment by MillionLights** [comment]
Known and annoying bug.
Mouse buttons presses don't register at all or get stuck and it happens completely at random.
Also not every game is effected so its very hard to troubleshoot or even realize it's LBM.

maybe the next update will fix it. Fingers crossed
